constellations/orion/src/repos/migrations/000019_create_table_ask_for_help.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE ask_for_help
+(
+    id          int unsigned NOT NULL AUTO_INCREMENT,
+    created_at  datetime     NOT NULL,
+    updated_at  datetime     NOT NULL,
+    deleted_at  datetime,
+    title       text         NOT NULL,
```

Comment:
It would probably be better if this is `varchar(256)` instead of `text` to save space.